### ISSUE TITLE
[ic-togggle-button] - loading variant of toggle button looks the same when checked and unchecked in high contrast mode

### ISSUE THREAD

**Issue by MI6-286** [mi6/ic-ui-kit#2367]
## Summary 
In high contrast mode, there is no difference in the appearance of a checked toggle button with loading prop and an unchecked toggle button with loading prop. In the image below, the toggle button on the right has the toggleChecked prop.
![image](https://github.com/user-attachments/assets/935e9109-d045-4e96-8fba-e8b8aabf8407)

## 💬 Description 
The appearance of the toggle button should be different when checked, in the same way as when not in high contrast mode. In the image below, the toggle button on the right has the toggleChecked prop.
![image](https://github.com/user-attachments/assets/f414d71f-6e5c-46e9-9ae4-05cd58976418)

## 💰 Use value 
Having a different appearance for a checked toggle button will mean that it is clear to end-users when a toggle button is checked.

**Comment by MI6-255** [comment]
Consider as part of the ticket how we show this in high contrast mode

**Comment by gd2910** [comment]
Use `cyan` as per the toggle checked when **not** in a loading state
